src/common/src/config.rs

```diff
@@ -1304,6 +1319,9 @@ pub mod default {
     }
 
     pub mod developer {
+        pub fn meta_cached_traces_num() -> u32 {
+            256
```

Comment:
Does this mean 256 lines of logs (i.e. tracing events)? Is it too small? Wondering how many logs in one epoch on average 🤔

Reply:
This means 256 trace(-root)s, which typically records the whole processing tree of 256 epochs or 256 queries.

Reply:
When I was working on suppressing logs, sometimes the log (before being found and suppressed by us) can be written millions of times. Is it possible to avoid OOM in that cases?

Reply:
Yeah. It's possible. 😕

Reply:
Added memory usage limit.

https://github.com/risingwavelabs/risingwave/pull/14220#discussion_r1440281137